### PR TITLE
Only include the populate_index and data

### PR DESCRIPTION
**Pull request by clintfred**
TODO
* [x] copy content from the website "try encrypted search" into the README
* [x] work out a good way to bring up elasticsearch and cloaked search (docker compose?)
* [x] make this repo public

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,89 @@
...
 
-Then play with the shell scripts to see what happens with commands like this:
+### (optional) Look at an encrypted index
 
+_Note that we are making this request directly to Elasticsearch (port 9200) so we can see what's actually stored._
+
+Let's get all the documents belonging to `tenant-1` and see what's in the index!
+
+```bash
+curl -s -G --data-urlencode "q=tenant_id:\"tenant-1\"" localhost:9200/try_cloaked_search/_search | jq
```
Comment (giarc3) [inline README.md:50]: Should this query (and all our other sample queries) use the `+tenant_id` like we're going to require?

**Review by giarc3 — CHANGES_REQUESTED** [review]
I was only able to get this to work after stopping my existing elasticsearch that was bound to port 9200. Should that be mentioned or be an optional step?

File: README.md (hunk 1 above)
Comment (clintfred) [reply README.md:52]: yep. added.

**Review by giarc3 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,54 +1,91 @@
-# Try Cloaked Search
+## Try Cloaked Search (in ~5 Minutes)
```
Comment (giarc3) [inline README.md:1]: What was the reasoning for bumping this down to `##`?

**Pull request merged**